src/pages/beta/[viewType]/curriculum/index.tsx

```diff
@@ -59,96 +72,102 @@ const CurriculumHomePage: NextPage<CurriculumHomePageProps> = (props) => {
+            $mb={80}
+            $pl={[16, 0]}
+            $pr={[16, 0]}
+            $mt={80}
+          >
+            <Box>
+              <Cover
+                $width={[320, 340, 480]}
+                $height={[300, 320, 450]}
+                $position={"relative"}
+              >
+                <Illustration
+                  noCrop
+                  sizes={getSizes([500, 800])}
+                  slug="curriculum-approach"
+                  $objectFit="contain"
+                  $objectPosition={"center"}
+                  fill
+                />
+              </Cover>
+            </Box>
+
+            <Box $height={"100%"}>
+              <Heading tag="h2" $font={["heading-5", "heading-4"]} $mt={12}>
+                Our approach to curriculum
+              </Heading>
+              <P $mv={[16, 24]} $font={"body-1"}>
+                Every school’s approach to curriculum design is different.
+                Whether you are starting from scratch or refreshing your
+                existing curricula, it can be helpful to see examples to inspire
+                your own. Explore our curriculum sequences and learn about the
+                six curriculum principles that guide our lesson and curriculum
+                design. See how we have:
+              </P>
+              <UL $mb={24} $font={"list-item-1"}>
+                <LI>
+                  Sequenced subject units across year groups, including
+                  alternative sequences
+                </LI>
+                <LI>Built coherence through the curricula, using threads</LI>
+                <LI>
+                  Summarised the core content pupils will learn, in our
+                  curriculum statements
+                </LI>
+              </UL>
+              <Typography $font={"heading-7"} $mb={12}>
+                <OakLink
+                  page={"webinar-index"}
+                  $display={"flex"}
+                  $alignItems={"center"}
                 >
-                  <CardLink
-                    page="blog-single"
-                    blogSlug="tbd-link-to-relevant-page"
-                  >
-                    <Heading tag={"h3"} $font={"heading-light-7"} $mb={16}>
-                      Resource
-                    </Heading>
-                    <P>How to design a unit of study</P>
-                  </CardLink>
-                </Card>
-              </GridArea>
-            </Grid>
-          </Box>
+                  Read more about our approach
+                  <Icon name={"chevron-right"} />
+                </OakLink>
+              </Typography>
+            </Box>
+          </Flex>
+          <Flex
+            $background={"grey1"}
+            $position={"relative"}
+            $minHeight={812}
+            $ml={[0, 12, 0]}
+            $mr={[0, 12, 0]}
+          >
+            <Box $pl={[24, 48]} $pr={[24, 48]} $pt={48} $mb={[48, 0]}>
+              <Heading tag="h3" $font={"heading-4"} $mb={24}>
```

Comment:
This should be a H2 so it's an equivalent heading to `Our approach to curriculum` and higher in the hierarchy than the blog post titles.

Reply:
✅